.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules/*
 !node_modules/helpers/
 node_modules/helpers/**/store/
+!node_modules/messages
```

Comment:
Lines 4-5 may be unnecessary. We already ignore all directories inside of `helpers/` as well as any occurrence of `store` inside of `helpers/`. If you agree you can move "messages" into "helpers" (& rename into @<!-- -->messages to respect naming conventions suggested here: https://github.com/virlancristian/shaorma-bot/pull/2#issuecomment-1738132162.

Reply:
Done, moved all the modules responsible for message interaction into helpers/@messages. Also, I separated helper functions for the /spam command into helpers/@spam. Check commit [49bdf65](https://github.com/virlancristian/shaorma-bot/pull/2/commits/49bdf657c1bd7219041c25e907c500f9c4648aa9)